transcription/transcription.go

```diff
@@ -29,3 +32,25 @@ func msgHeaders(from string, to []string, subject string) string {
 	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
 	return strings.Join(msgHeaders, "\r\n")
 }
+
+// Transcribe a given file using a Sphinx jar. The fileName should be in
+// "name.wav" format and the jarName should be in "name.jar" format.
+func startTranscription(fileName string, jarName string) error {
+
+	cmd := exec.Command("java", "-jar", jarName, fileName)
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		return err
+	}
+	if err := cmd.Start(); err != nil {
+		return err
+	}
+	//Currently copying output to Stdout
```

Comment:
Sorry that I'm busy, I'll give this a look later this week!

-- 
Benjamin Sandler
University of Pennsylvania | Class of 2018
Candidate for BS in Computer Science & BA in Cognitive Science
954.478.6689 | ben1sandler@gmail.com

On Wed, Mar 16, 2016 at 12:05 AM Yoni Nachmany <notifications@github.com>
wrote:

> In transcription/transcription.go
> <https://github.com/hack4impact/audio-transcription-service/pull/26#discussion_r56279362>
> :
>
> > @@ -29,3 +32,25 @@ func msgHeaders(from string, to []string, subject string) string {
> >  	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
> >  	return strings.Join(msgHeaders, "\r\n")
> >  }
> > +
> > +// Transcribe a given file using a Sphinx jar. The fileName should be in
> > +// "name.wav" format and the jarName should be in "name.jar" format.
> > +func startTranscription(fileName string, jarName string) error {
> > +
> > +	cmd := exec.Command("java", "-jar", jarName, fileName)
> > +	stdout, err := cmd.StdoutPipe()
> > +	if err != nil {
> > +		return err
> > +	}
> > +	if err := cmd.Start(); err != nil {
> > +		return err
> > +	}
> > +	//Currently copying output to Stdout
>
> Please add space between // and comment.
>
> Is it valid to copy output to Stdout? Also, good to note where you plan to
> copy output to in future.
>
> —
> You are receiving this because you were mentioned.
>
> Reply to this email directly or view it on GitHub
> <https://github.com/hack4impact/audio-transcription-service/pull/26/files/7fa558818e4d4e0ba9aebb353c09f59d89224205#r56279362>
>